contentcuration/contentcuration/frontend/channelEdit/views/ResourcePanel.vue

```diff
@@ -311,18 +387,24 @@
       },
     },
     watch: {
-      node() {
+      // Listen to node id specifically to avoid recursvie call to watcher,
+      // but still get updated properly if need to wait for node to be loaded
+      'node.id'() {
```

Comment:
Might be a bit more intuitive to use `nodeId`

Reply:
I tried that, but ran into the issue where the nodeId was available, but the actual node wasn't, leading to some issues